scripts/utils/integration-tests

```diff
@@ -51,8 +53,10 @@ PLUGINS_PATH=${PLUGINS_PATH:-''}
 # OUTPUT_PATH: Directory where results are saved (e.g. "some/path/to/Testfile-cy.js-1")
 function executeCypress {
   npm run --silent cypress -- \
+    ${HEADED:+--headed} \
```

Comment:
🎨 not strictly necessary, but I'd always quote strings.

Reply:
@juliangieseke do you maybe know how to pass those flags in a better way? Currently one can write `run integration tests Spec dima julian` and still have both `--headed` and `--no-exit` :D

Reply:
would `run integration tests Spec -- --headed --no-exit` work 🤔

Reply:
https://jonalmeida.com/posts/2013/05/26/different-ways-to-implement-flags-in-bash/

Reply:
dealing with arguments in bash is a common issue (say: complicated) and one of the reasons a lot of scripts rely on ENV_VARS… various implementations of `getopt(s)` are buggy in some ways and you still have to write a lot of gluecode :/